src/sql/workbench/electron-browser/modelComponents/diffeditor.component.ts

```diff
@@ -25,6 +24,16 @@ import { TextDiffEditor } from 'vs/workbench/browser/parts/editor/textDiffEditor
 import { DiffEditorInput } from 'vs/workbench/common/editor/diffEditorInput';
 import { TextDiffEditorModel } from 'vs/workbench/common/editor/textDiffEditorModel';
 import { CancellationTokenSource } from 'vs/base/common/cancellation';
+import { ITextModelService } from 'vs/editor/common/services/resolverService';
+import { ITextModel } from 'vs/editor/common/model';
+
+class ServiceAccessor {
+	constructor(
```

Comment:
Whats the reason for this abstraction? why not just inject the services into the class?

Reply:
I was just copying what was in a test. I fixed this to just inject the service.